Backend/auction-app/src/main/java/com/example/auctionapp/security/MyUserDetailsService.java

```diff
@@ -0,0 +1,34 @@
+package com.example.auctionapp.security;
+
+
+import com.example.auctionapp.model.User;
+import com.example.auctionapp.repository.BaseRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Service
+public class MyUserDetailsService implements UserDetailsService {
```

Comment:
`MyUserDetailsService` is a bad name, `RepositoryAwareUserDetailsService` is better.